charts/jaeger/templates/_helpers.tpl

```diff
@@ -419,3 +419,27 @@ Cassandra or Elasticsearch related command line options depending on which is us
 {{- include "elasticsearch.cmdArgs" . -}}
 {{- end -}}
 {{- end -}}
+
+{/*
+Return the appropriate apiVersion for ingress.
```

Comment:
@mfuxi my bad for missing this, but can you remove these changes in helpers.tpl file completely.
And for check in side the ingress.yaml files, just use `"common.ingress.supportsIngressClassname"` to set the ingressClassName field.